specification/schemas/LocalitiesAutocompleteResponse.yml

```diff
@@ -29,9 +29,15 @@ properties:
     example: "London, UK"
     deprecated: true
   location:
-    title: LatLngLiteral
```

Comment:
`LatLngLiteral` should also also be used in autocomplete bias parameter if I'm not mistaken, so imo this modif should also be applied there, or LatLngLiteral should be kept in both to factorize doc

Reply:
the issue is, as this field is deprecated here; we can't use `$ref: "./LatLngLiteral.yml" because it will not consider the field deprecated.

Reply:
if LatLngLiteral is used in autocomplete bias, it should be mentioned in this OpenAPI Spec